scripts/firebase-admin/updateGeneralCourt.ts

```diff
@@ -0,0 +1,92 @@
+import { Script } from "./types"
+import { currentGeneralCourt } from "../../functions/src/shared"
+
+export const script: Script = async ({ db, args }) => {
```

Comment:
One nit: could we add a `dryRun` arg here, and only actually make the updates if it's set to `true`? This would make it easier for us to first run this script without making any changes just to see that the proposed changes look correct. 

There's other examples of us using args in other scripts, like here: https://github.com/codeforboston/maple/blob/935407cc3434398b61622aa3c21f8c695dec1818/scripts/firebase-admin/backfillNextDigestAt.ts#L16